src/parser/priest/holy/modules/talents/90/Halo.tsx

```diff
@@ -43,7 +45,7 @@ class Halo extends Analyzer {
 
   statistic() {
     return (
-      <TalentStatisticBox
+      <Statistic
```

Comment:
Im not sure what end-goal is, but you can provide
`category={STATISTIC_CATEGORY.TALENT}` if you want to keep talents grouped up